src/models/query_models.rs

```diff
@@ -0,0 +1,118 @@
+use mongodb::bson::doc;
+use std::{collections::HashMap, fmt};
+
+#[derive(serde::Deserialize, Clone)]
+pub struct InfoQueryBody {
+    pub command: Option<String>,
+    pub catalogs: Option<Vec<String>>,
+}
+
+#[derive(serde::Deserialize, Clone)]
+pub enum Unit {
```

Comment:
could use some of serdes attributes so we can also accept shorthands and lowercased versions. To do in another PR